### DISCUSSION TITLE
Lesson 3: new address[](0)

### DISCUSSION THREAD

**Discussion by EWCunha** [smartcontractkit/full-blockchain-solidity-course-py#1252] [Q&A] [answered]
Hi guys! I don't understand what is the `(0)` after `new address[]` (i.e., `new address[](0)`). I mean, wouldn't that be enough to write only `funds = new address[]` (instead of `funds = new address[](0)`) to reset the `funds` array? What does the `(0)` actually do?

Thank you!

**Comment by cromewar** [comment, marked as answer]
Hellor @EWCunha it refers to the size of the array:
![image](https://user-images.githubusercontent.com/20199180/158026817-49f6a0b9-a022-4388-96e6-7270c0793e31.png)

**Reply by EWCunha** [reply]
Thank you very much!
